engine/step/step.go

```diff
@@ -32,6 +32,7 @@ const (
 const (
 	StateAny           = "ANY" // wildcard
 	StateTODO          = "TODO"
+	StateInLoop        = "IN_LOOP"
```

Comment:
Personal preference, but I'd rename it to `StateLooping`, with value `LOOPING` for consistency with existing states.